.github/ISSUE_TEMPLATE/compiler_impact_report.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Compiler Impact Report
```

Comment:
"Compiler Impact" -> "Compiler Support" may be more specific (along with file name)

Reply:
Done in 72eae4b.